core/local/index.js

```diff
@@ -124,15 +132,17 @@ module.exports = class Local /*:: implements Side */ {
 
   /* Helpers */
 
-  // Return a function that will update last modification date
-  // and does a chmod +x if the file is executable
-  //
-  // Note: UNIX has 3 timestamps for a file/folder:
-  // - atime for last access
-  // - ctime for change (metadata or content)
-  // - utime for update (content only)
-  // This function updates utime and ctime according to the last
-  // modification date.
+  /**
+   * Return a function that will update last modification date
+   * and does a chmod +x if the file is executable
+   *
+   * Note: UNIX has 3 timestamps for a file/folder:
+   * - atime for last access
+   * - ctime for change (metadata or content)
+   * - utime for update (content only)
```

Comment:
is it `utime` or `mtime`?

Reply:
Actually this comment is not accurate at all: `utimes` changes `atime`, not `ctime` :man_facepalming: 

But I don't think I changed anything here (except moving stuff around).
So let's fix it in another PR.